apps/dashboard/src/app/applications.ts

```diff
@@ -265,5 +269,16 @@ export const applications: IApplications = <const>{
       } as IModelAssessmentDataSet
     },
     versions: { "1": 1, "2:Static-View": 2 }
+  },
+  modelAssessmentVision: {
+    datasets: {
+      visionModelExplanationData: {
+        classDimension: 3,
+        dataset: emotion,
+        modelExplanationData: [emotionModelExplanationData],
```

Comment:
emotionModelExplanationData?  this looks wrong?

Reply:
Discussed, this can be removed. Also we talked about moving out the visionModelExplanationData properties into dataset instead.